ci/federalist-pipeline.yml

```diff
@@ -0,0 +1,307 @@
+---
+############################
+#  SHARED
+
+env-cf: &env-cf
+  CF_API: https://api.fr.cloud.gov
+  CF_USERNAME: ((production-cf-username))
+  CF_PASSWORD: ((production-cf-password))
+  CF_ORG: gsa-18f-federalist
+  CF_SPACE: production
+
+node-image: &node-image
+  platform: linux
+  image_resource:
+    type: docker-image
+    source:
+      repository: node
+      tag: 14
```

Comment:
[comment] Noticed we aren't super consistent with these versions - I think some of the others pipelines I was working on were at `16` - maybe in the future we can parameterize that as well?

Reply:
I noticed that too. Not sure the best way to manage those but we'll need to move up to 16 which active LTS or to 18 which is current and soon to be active LTS.